rfcs/text/semantic-release.md

```diff
@@ -0,0 +1,257 @@
+
+# Adoption strategy
+
+This RFC PR includes all changes necessary, if this RFC is accepted when it is merged it will immediately start
+releasing from the following branches `['+([0-9])?(.{+([0-9]),x}).x', 'master', 'next', 'next-major', {name: 'beta', prerelease: true}, {name: 'alpha', prerelease: true}]`  
+[[Source]](https://github.com/semantic-release/semantic-release/blob/916c2685c57f3490fb1e50afbf72ea8dce11e188/docs/usage/configuration.md#branches)
+
+Adding [`probot/semantic-pull-requests`](https://probot.github.io/apps/semantic-pull-requests/) is a manual step and
+will have to be performed by a workspace administrator.
+
+# How we teach this
+
+The proposed tooling is well established in the JavaScript community. For carbon developers
+ can use `npm run commit` to launch [`commitizen`](https://github.com/commitizen/cz-cli) which will guide them through the process.
+![](./commitizen.svg)
+
+| Type           | Description                                                                                                 | Release Type |
+|----------------|-------------------------------------------------------------------------------------------------------------|--------------|
+| `feat:`        | A new feature                                                                                               | minor        |
+| `fix:`         | A bug fix                                                                                                   | patch        |
+| `docs:`        | Documentation only changes                                                                                  | :x:          |
+| `style:`       | Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)      | :x:          |
+| `refactor:`    | A code change that neither fixes a bug nor adds a feature                                                   | :x:          |
+| `perf:`        | A code change that improves performance                                                                     | patch        |
+| `test:`        | Adding missing tests or correcting existing tests                                                           | :x:          |
+| `build:`       | Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)         | :x:          |
+| `ci:`          | Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs) | :x:          |
+| `chore:`       | Other changes that don't modify src or test files                                                           | :x:          |
+| `revert:`      | Reverts a previous commit                                                                                   | patch        |
+
+This behaviour is defined in [@semantic-release/commit-analyzer](https://github.com/semantic-release/commit-analyzer/blob/95a21dc7b09b7bff753d3d4a00e684b69a81df8c/lib/default-release-rules.js).
```

Comment:
```suggestion
This behaviour is defined in [@semantic-release/commit-analyzer](https://github.com/semantic-release/commit-analyzer/blob/v7.0.0-beta.7/lib/default-release-rules.js).
```

Reply:
Because tags can change I would like to keep this pointing at the specific commit.